rust/src/lib/nm/settings/ip.rs

```diff
@@ -64,6 +64,7 @@ fn gen_nm_ipv4_setting(
     }
     nm_setting.method = Some(method);
     nm_setting.addresses = addresses;
+    nm_setting.forwarding = iface_ip.forwarding.map(|f| f as i32);
```

Comment:
trivial:

For my `as` is dangerous as has integer wrapping. Yes, `bool as i32` is safe, but better using `i32::from` which is safer.